components/leaflet-maps/ClustersMap.vue

```diff
@@ -165,6 +176,17 @@
 
        return locationsWithValue
      },
+     getStandardsWithValue: function (witch) {
```

Comment:
I would probably name this getStandardAttributesWithValues. Just so it is a bit clearer what the method is returning.